joplin/base/signals/janis_build_triggers.py

```diff
@@ -13,19 +13,48 @@
 from base.signals.aws_publish import get_http_request, create_build_aws
 from base.signals.netlify_publish import netlify_publish
 
+import graphene
+
 import logging
 logger = logging.getLogger(__name__)
 
 JANIS_SLUG_URL = settings.JANIS_SLUG_URL
 
 
+# Copying this stuff in there to get data without errors
+# Get a page global_id from a page chooser node
+# Works for any content_type defined in content_type_map
+def get_graphql_id(page):
+    # copypastaing this because importing was being messy
```

Comment:
I've run into this problem too. Let's find a solution for handling messy imports.